src/mudlet-lua/tests/MudletBusted_spec.lua

```diff
@@ -0,0 +1,9 @@
+describe("Mudlet Busted sanity check", function()
+    it("runs a simple assert", function()
+        assert.truthy("Not nil or false.")
+      end)
+    it("checks that it has access to mudlet environment functions", function()
+        assert.truthy(string.find(getMudletHomeDir(), "udlet"))
```

Comment:
```suggestion
        assert.truthy(string.find(getMudletHomeDir(), "mudlet"))
```

?

Reply:
In case of different capitalization. `udlet` will catch both `Mudlet` and `mudlet`

Reply:
Oh. Let's go with `[mM]udlet` - more self-explanatory